src/user/user.service.ts

```diff
@@ -29,6 +29,11 @@ export class UserService {
         return this.model.exists({ _id: id });
     }
 
+    // async isAdmin(id: string): Promise<boolean> {
+    //     let user = await this.model.findById(id).exec();
+    //     return user.role === UserRole.Admin
+    // }
+
```

Comment:
This `user.service.ts` file does not need to be change. Simply discard changes before commit.

Reply:
Actually , I'm was going to remove that before things get done.

Reply:
You do not just stage all files. Select and stage only files that are affected, because this file is completely unrelated.